main.py

```diff
@@ -37,9 +39,23 @@ def main():
             print("Cleaning up namespace %s" % namespace.metadata.name)
             # real cleanup will be enable later once it's tested
             v1api.delete_namespace(namespace.metadata.name, client.V1DeleteOptions())
+            cleaned += 1
 
     print("Finish clean up script")
-
+    return cleaned
 
 if __name__ == '__main__':
-    main()
+    start = time.time()
+    eventdict = { 'name' : 'msftkube', 'time' : start, 'eventtype' = 'json'}
+    try
+        namespacescleaned = main()
```

Comment:
I think it would be nice to leave main() as the top level function and rename what was previously main to a different function (e.g. main() is still what gets executed but then it calls cleanupNamespace() or something like that)

Reply:
yep agree